vnext/Scripts/msbuildeval.ps1

```diff
@@ -0,0 +1,73 @@
+# Evaluate an MSBuild property name
+param(
+    [Parameter(Mandatory=$true)]
+    [String]$SolutionFile,
+    [Parameter(Mandatory=$true)]
+    [String]$ProjectFile,
+    [Parameter(Mandatory=$true)]
+    [String]$PropertyName
+)
+
+function Get-MSBuildPath {
+    $vsWhere = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe"
+    if (!(Test-Path $vsWhere)) {
+        throw "Unable to find vswhere.exe."
+    }
+    $vsPath = & $vsWhere -version 16.5 -property installationPath;
+    return "$vsPath\MSBuild\Current\Bin\";
+}
```

Comment:
The CLI has some extra logic compared to this to determine which copy of MSBuild to use. E.g. it will only consider installations with certain packages, and has special handling of prereleases.

Consider allowing the CLI to explicitly pass its own resolved MSBuild path instead of trying to replicate the logic.

Reply:
Yeah, this logic isn't great, but I wanted to start with a "standalone" PS solution, and our CLI logic for MSBuild is all buried in JS.